pepper-apis/src/main/java/org/broadinstitute/ddp/util/StudyDataLoaderMain.java

```diff
@@ -653,17 +662,30 @@ private void processParticipant(String studyGuid, Map<String, JsonElement> sourc
                     hasFollowup = (sourceData.get("followupsurvey") != null && !sourceData.get("followupsurvey").isJsonNull());
                     hasFollowupConsents = (sourceData.get("followupconsentsurvey") != null
                             && sourceData.get("followupconsentsurvey").getAsJsonArray().size() > 0);
+                    hasPrionConsent = (sourceData.get("ConsentSurvey") != null && !sourceData.get("ConsentSurvey").isJsonNull());
+                    boolean hasSomeMedical = sourceData.get("MedicalSurvey") != null && !sourceData.get(
+                            "MedicalSurvey").isJsonNull();
+                    hasMedical = (hasSomeMedical && hasPrionConsent && (completeStatusElement =
+                            sourceData.get("ConsentSurvey").getAsJsonObject().get("complete_status")) != null
+                            && !completeStatusElement.isJsonNull() && "1".equals(completeStatusElement.getAsString()));
+
+                    if (hasSomeMedical && !hasMedical) {
+                        LOG.warn("Not loading medical questionnaire for participant (altpid " + altpid + ", email "
+                                + emailAddress + ") because consent survey has not been completed.");
+                    }
 
                     var answerDao = handle.attach(AnswerDao.class);
 
                     //create prequal
-                    dataLoader.createPrequal(handle,
-                            userGuid, studyId,
-                            createdAt,
-                            jdbiActivity,
-                            activityInstanceDao,
-                            activityInstanceStatusDao,
-                            answerDao);
+                    if (!"PRION".equals(studyGuid)) {
```

Comment:
May be we can pass a flag like skipPrequal or noPrequal and use it rather than hardcoding for a study